### ISSUE TITLE
[Huawei_analytics] FATAL EXCEPTION when start the app

### ISSUE THREAD

**Issue by mustafa-707** [HMS-Core/hms-flutter-plugin#141]
**Description**
I just implement the huawei_analytics plugin to my app , and everything is fine but with some android devices which it have GMS and HMS services we notice there's an exception appears on our crachlytics tool in release mode 

```
E/AndroidRuntime(26270): FATAL EXCEPTION: main
E/AndroidRuntime(26270): Process: com.ex.app, PID: 26270
E/AndroidRuntime(26270): java.lang.NullPointerException: Attempt to read from field 'java.lang.String com.huawei.hms.analytics.ar.ghi' on a null object reference
E/AndroidRuntime(26270):        at com.huawei.hms.analytics.ijk.klm(Unknown Source:14)
E/AndroidRuntime(26270):        at com.huawei.hms.analytics.a$2.run(Unknown Source:72)
E/AndroidRuntime(26270):        at android.os.Handler.handleCallback(Handler.java:809)
E/AndroidRuntime(26270):        at android.os.Handler.dispatchMessage(Handler.java:102)
E/AndroidRuntime(26270):        at android.os.Looper.loop(Looper.java:166)
E/AndroidRuntime(26270):        at android.app.ActivityThread.main(ActivityThread.java:7555)
E/AndroidRuntime(26270):        at java.lang.reflect.Method.invoke(Native Method)
E/AndroidRuntime(26270):        at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:469)
E/AndroidRuntime(26270):        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:963)
```

**Comment by furkansarihan** [comment]
Hi @mustafa-707,

This exception resolved with Huawei Analytics SDK 6.3.0.303.

We'll update this package with new Analytics SDK version in our next major release.  Keep an eye on [pub.dev](https://pub.dev/packages/huawei_analytics) for updates.

### For Quick Fix;

You can fork the repo and change the Analytics SDK version at line 44 in `android/build.gradle` to 6.3.0.303.

``` gradle
dependencies {
    implementation 'com.huawei.hms:hianalytics:6.3.0.303'
}
```

**Comment by mustafa-707** [comment]
thank you , I will do
